cookbooks/aws-parallelcluster-environment/kitchen.environment-recipes.yml

```diff
@@ -0,0 +1,88 @@
+# Validates config recipes
```

Comment:
This file name is out of the standard convention. 

We were trying to split the tests in two files: <workstream>-config.yml and <workstream>-install.yml containing both tests for recipes and resources.

This is probably the reason why you have to redefine cluster attibutes, verifiers etc.

Can we move the controls you defined here, into the <workstream>-config.yml?

Reply:
I was trying to replicate what is in the top level validate-config to see similar behavior to what runs in our system tests since there are some differences there from what is in the other -install and -config tests.  But I realized after creating this that the tests won't work anyway since we need file systems to attach.  I will remove this for now